lib/mixins/property-effects.js

```diff
@@ -964,6 +964,15 @@ function parseArg(rawArg) {
   return a;
 }
 
+function getArgValue(data, props, path) {
+  let value = get(data, path);
+  // when data is not stored e.g. `splices`, get the changed value
```

Comment:
Let's add a comment and/or link to an issue here about this being technically incorrect since it can cause an `undefined` path to return a stale value.